sway-book/src/documentation/language/annotations/attributes/index.md

```diff
@@ -0,0 +1,3 @@
+# Attributes
+
+An attribute is a metadatum which provides some additional functionality.
```

Comment:
There should be another attribute for the test attribute recently added.

Reply:
cc @mitchmindtree for that absolutely incredible PR